kalite/store/models.py

```diff
@@ -98,12 +97,11 @@ def handle_exam_unset(sender, **kwargs):
         for testlog in testlogs:
             facility_user = testlog.user
             facility = facility_user.facility
-            unit_id = get_current_unit_settings_value(facility.id)
             ds = load_dynamic_settings(user=facility_user)
             if ds["student_testing"].turn_on_points_for_practice_exams:
-                transaction_log, created = StoreTransactionLog.objects.get_or_create(user=testlog.user, context_id=unit_id, context_type="output_condition", item="gift_card")
+                transaction_log, created = StoreTransactionLog.objects.get_or_create(user=testlog.user, context_id=0, context_type="output_condition", item="gift_card")
                 try:
-                    transaction_log.value = int(round(settings.UNIT_POINTS * float(testlog.total_correct)/testlog.total_number))
+                    transaction_log.value = int(round(settings.POINTS * float(testlog.total_correct)/testlog.total_number))
```

Comment:
Actually, we should just not have any points here, I think - this was only required for the output condition of the RCT.